include/Core/Base/Mixin/Fetcher.php

```diff
@@ -22,6 +22,10 @@ protected function get_post_types() {
 		return bd_get_post_types();
 	}
 
+	protected function get_taxonomies() {
```

Comment:
Add PHPDoc block comment for this method.

Basically if the method is not a inherited method then you have to add comments.

Reply:
Fixed